### ISSUE TITLE
Setup peril for goby-lang project management

### ISSUE THREAD

**Issue by SD10** [goby-lang/goby#637]
### Proposal: Setup peril for goby-lang project management
I think that [peril](https://github.com/danger/peril) could be a valuable resource for automating work around goby-lang repositories.

#### Why peril?

You may be familiar with a tool called [danger](https://github.com/danger/danger) by the same organization. Danger hooks into your CI environment and allows you to automate code review on pull requests by enforcing rules customizable through a [Dangerfile](https://github.com/Moya/Moya/blob/master/Dangerfile)

You can:
- Enforce CHANGELOGs
- Enforce links to Trello/JIRA in PR/MR bodies
- Enforce using descriptive labels
- Look out for common anti-patterns
- Highlight interesting build artifacts
- Give specific files extra focus

Peril is very similar to danger, however, it does not have the limitation of being coupled with your continuous integration service. 

Instead, peril is capable of watching any GitHub event hook across all repos of an organization. This means we can automate tasks around issues, pull requests, and other events such as comments.

We could potentially build a bot to:
- Auto label issues
- Review pull requests
- Watch for comments from maintainers to trigger other events such as CI builds 

### TL;DR
I think we can use peril to bootstrap a pretty powerful goby-lang bot in the future.

**Comment by st0012** [comment]
@SD10 good proposal! thanks!
I think there're at least two things we can do via the bot:
1. automatically assign issue to maintainers
2. Run benchmark and comment the result on PR

**Comment by st0012** [comment]
@SD10 sorry but I'm going to drop the plan of this for now. we can revisit this in the future
